### PR TITLE
[CMS-578] Initial migrate to build tools docs page.

### PR DESCRIPTION
**Pull request by kporras07**
Closes CMS-578

## Summary

**[Migrating a Drupal with Composer site to a Build Tools site](https://pr-7158--pantheon-docs.my.pantheonfrontend.website/docs/guides/drupal-9-to-build-tools)** - Add new migrate Drupal 9 to build tools page

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] Docs review and updates from the docs team

**Release**:
- [X] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7158](https://pr-7158--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `56.99999999999999` | `64` | `7` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-7158` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/34545/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/34545/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7158--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/guides/drupal-9-to-build-tools.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+  ```bash{promptUser:user}
+  mkdir -p $DESTINATION/web/themes/custom
+  cp -r $SOURCE/web/themes/custom $DESTINATION/web/themes/custom
+  # From $DESTINATION:
+  git add web/themes/
+  git commit -m "Copy custom themes"
+  ```
+
+1. Use the above commands to move custom code (if any) to your new site.
```
Comment (greg-1-anderson) [inline source/content/guides/drupal-9-to-build-tools.md:153]: This line shouldn't be a bullet item.

**Review by greg-1-anderson — APPROVED** [review]

**Comment by whitneymeredith** [comment]
[Preview](https://pr-7158--pantheon-docs.my.pantheonfrontend.website/docs/guides/drupal-9-to-build-tools)

**Review by whitneymeredith — APPROVED** [review]
From a Docs perspective, this is good to go.

**Pull request merged**